extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -272,6 +276,24 @@ export class GoogleSubscriptionsPlatform {
     // Install styles.
     installStylesForDoc(ampdoc, CSS, () => {}, false, TAG);
 
+    /** @private @const {boolean} */
+    this.meteringEnable_ = !!this.serviceConfig_['enableMetering'] || false;
```

Comment:
Could this variable be named `enableMetering` to be consistent with `enableLAA`?

Reply:
Also, could this be a local variable?